cli/dcoscli/data/config-schema/core.json

```diff
@@ -9,6 +9,7 @@
     },
     "dcos_url": {
       "type": "string",
+      "pattern": "^(?:(?:https?)://)(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\\.)+(?:[a-zA-Z]{2,6}\\.?|[a-zA-Z0-9-]{2,}\\.?)|localhost|\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})(?::\\d+)?(?:/?|[/?]\\S+)$",
```

Comment:
This pattern appears to fail for URLs that have only one part (apart from localhost):
```
➜  cli git:(dcos-1625) ✗ dcos config set core.dcos_url http://dcos-1234
Value "http://dcos-1234" does not meet the requirements for "dcos_url" property
```